mets.md

```diff
@@ -159,7 +170,7 @@ Always use URL. If it's a local file, prefix absolute path with `file://`.
 
 ```xml
 <mets:fileGrp USE="OCR-D-SEG-BLOCK">
-    <mets:file ID="OCR-D-SEG-PAGE_0001" GROUPID="OCR-D-IMG_0001" MIMETYPE="application/vnd.prima.page+xml">
+    <mets:file ID="OCR-D-SEG-PAGE_0001" MIMETYPE="application/vnd.prima.page+xml">
```

Comment:
Did we overlook that? Doesn't it have to be OCR-D-SEG-BLOCK_0001 instead of OCR-D-SEG-PAGE_0001 as file@ID?

Reply:
I think you're right.